File: java/viagem.java
public class viagem {
    public static void main(String[] args) {
        double precoCombustivel = 6.17;
        double precoGasolina = 6.17;
        double precoEtanol = 4.40;

        int kmPorLitro = 10;
        int distanciaEmKm = 100;

        String tipoCombustivel = "Etanol";

        double litroConsumido = (double) distanciaEmKm / kmPorLitro;
        double valorGasto = litroConsumido * precoCombustivel;

        System.out.println("Você vai gastar R$ " + valorGasto + " para percorrer " + distanciaEmKm + "Km");

        double valorViagem;

        if (tipoCombustivel.equals("Gasolina")) {
            valorViagem = precoGasolina * distanciaEmKm;
            System.out.println("Com Gasolina vai custar " + String.format("%.2f", valorViagem));
        } else if (tipoCombustivel.equals("Etanol")) {
            valorViagem = precoEtanol * distanciaEmKm;
            System.out.println("Com Etanol vai custar " + String.format("%.2f", valorViagem));
        } else {
            System.out.println("Não identificamos o seu tipo de combustível");
        }
    }
}

File: python/viagem.py
preco_combustivel = 6.17
preco_gasolina = 6.17
preco_etanol = 4.40

km_por_litro = 10
distancia_em_km = 100

tipo_combustivel = 'Etanol'

litros_consumidos = distancia_em_km / km_por_litro
valor_gasto = litros_consumidos * preco_combustivel

print(f'Você vai gastar R$ {valor_gasto:.2f} para percorrer {distancia_em_km}Km')

if tipo_combustivel == 'Gasolina':
    valor_viagem = preco_gasolina * distancia_em_km
    print(f'Com Gasolina vai custar {valor_viagem:.2f}')
elif tipo_combustivel == 'Etanol':
    valor_viagem = preco_etanol * distancia_em_km
    print(f'Com Etanol vai custar {valor_viagem:.2f}')
else:
    print('Não identificamos o seu tipo de combustível')
